curriculum/challenges/english/07-scientific-computing-with-python/learn-interfaces-by-building-an-equation-solver/662bdd364bf2cde1487922a9.md

```diff
@@ -0,0 +1,43 @@
+---
+id: 662bdd364bf2cde1487922a9
+title: Step 5
+challengeType: 20
+dashedName: step-5
+---
+
+# --description--
+
+Unlike other programming languages, Python does not implement interfaces in its core language, but Python standard library allows you to define interfaces in a simple way.
+
+For this project, you'll use utilities from the `abc` module. So, go at the top of your code and import `ABC` from the `abc` module.
```

Comment:
It is better to mention what `ABC` is in this step rather than the next step.

Reply:
Also, the editable region is already at the top so the user shouldn't be instructed to go to the top.